src/Singularity/Extensions/ContainerBuilderExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Singularity
+{
+    /// <summary>
+    /// Extensions for <see cref="ContainerBuilder"/>.
+    /// </summary>
+    public static class ContainerBuilderExtensions
+    {
+
+        /// <summary>
+        /// Registers services defined in an <see cref="IModule"/>.
+        /// </summary>
+        /// <typeparam name="TModule"></typeparam>
+        public static void RegisterModule<TModule>(this ContainerBuilder containerBuilder)
```

Comment:
Why not just put the method on the ContainerBuilder itself? Having a separate extension class doesn't really add any value.

Reply:
I originally had this as a method on the ContainerBuilder class, but then my broken brain is like _You should totally make this into an extension method._ Not surprised that I went ahead and made it into an extension method.

I have just submitted a change to move this back to the the ContainerBuilder class.